partner_delivery_schedule/tests/test_partner_delivery_schedule.py

```diff
@@ -4,11 +4,10 @@
 from odoo.tests import Form, TransactionCase
 
 
-class TestStockPikingReturnRefundOption(TransactionCase):
-    @classmethod
-    def setUpClass(cls):
```

Comment:
Pls do not change this. Keep setUpClass and use SavepointCase instead.
Also, it would be nice to disable tracking if you don't need it: `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`

Reply:
Done, thanks!